lib/state/ui/reducer.ts

```diff
@@ -206,6 +208,20 @@ const openedTag: A.Reducer<T.TagHash | null> = (state = null, action) => {
   }
 };
 
+const showUntaggedNotes: A.Reducer<boolean> = (state = false, action) => {
+  switch (action.type) {
+    case 'SHOW_UNTAGGED_NOTES':
+      return true;
+    case 'SELECT_TRASH':
+    case 'SHOW_ALL_NOTES':
+    case 'OPEN_TAG':
+    case 'TRASH_TAG':
```

Comment:
why would trashing a tag close out the untagged view?

Reply:
Good point, removing that!

Reply:
No longer pertinent due to state refactor.